src/providers/ext.rs

```diff
@@ -14,7 +13,11 @@ pub trait InfoProviderExt: InfoProvider {
         self.token_info(address).await.ok()
     }
 
-    async fn addresses_to_address_ex(
+    async fn add_address_info_from_contract_info_or_empty(&self, address: &String) -> AddressEx {
```

Comment:
could we rename this to `to_address_ex_contracts` ? I suppose we don't have `to_address_ex_any_source` in the `ext`

Reply:
The point hwre is that it doesn't baul, and instead returns an addressex where only the address is populated. I would like to make this clear in zhe method name, as this was throwing me off in the original naming

Reply:
See above.